src/components/Header/Header.tsx

```diff
@@ -0,0 +1,22 @@
+import './Header.scss';
```

Comment:
I don't see any styles in [Header.scss](https://github.com/mate-academy/react_todo-app-loading-todos/pull/1214/files#diff-28618c5153a8687710c0971cf9036da8ed83f42f1b2f15274fb894f6028c7684), don't forget to move them to it